community/pmc-tool/README.md

```diff
@@ -0,0 +1,18 @@
+# Hello! It is a tool that can be run daily or weekly to analyze information about the open source community. It requests information from the Git API and generates a report on the latest state of the community.
+
+# Installation 
+git clone git@github.com/AibekYrysbekov/airflow
+
+# Create virtualenv venv and install requirements.txt
+pip3 install -r requirements.txt
```

Comment:
Let's make this more concise and correct

```suggestion
# Installation 

```sh
# Clone repository and open the directory
git clone git@github.com/AibekYrysbekov/airflow
cd airflow/community/pmc-tool

# Create virtualenv venv and install requirements.txt
pip3 install -r requirements.txt
```

## TODO

```